core/modules/user/user.module

```diff
@@ -453,6 +453,23 @@ function user_password($length = 10) {
   return $pass;
 }
 
+/**
+ * Checks if a user role has a permission.
+ *
+ * @param string $role_name
+ *   The role machine name.
+ * @param string $permission_name
+ *   The permission to check.
+ *
+ * @return boolean
```

Comment:
The [data type](https://docs.backdropcms.org/doc-standards#types) to use is `bool`.

Reply:
Thanks @kiamlaluno - Fixed